docs/reference/cat/nodeattrs.asciidoc

```diff
@@ -1,8 +1,77 @@
 [[cat-nodeattrs]]
 === cat nodeattrs
 
-The `nodeattrs` command shows custom node attributes.
-For example:
+Returns information about custom node attributes.
+
+[[cat-nodeattrs-api-request]]
+==== {api-request-title}
+
+`GET /_cat/nodeattrs`
+
+[[cat-nodeattrs-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=http-format]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-h]
++
+See <<cat-nodeattrs-api-columns>> for a list of available columns. See
+<<cat-nodeattrs-api-ex-headings>> for an example.
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=help]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=local]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=master-timeout]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-s]
```

Comment:
In other docs (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/current/certgen.html) I've put the short and long form of parameters separated by a comma (e.g. "-s, --silent", "-v, --verbose").  I notice that these docs use brackets (e.g. "s (sort)", "v (verbose)").  If possible, I think we should come to agreement on a style and use it as consistently as possible.

Reply:
In this case, this actually isn't the verbose or long form of the parameter. It's merely providing some explanatory text for the abbreviation.

I can remove those if you feel they're confusing.

Reply:
Ah, I though these were alternative keywords.  In my opinion, it is unclear.

Reply:
Thanks! Removed with b36a6f294e49d008f7b4472d9b9277c8aba84a15.